lib/definitions.php

```diff
@@ -760,6 +762,110 @@
 	EXISTINGASSIGNMENT_DELETE => trans('<!existingassignment>delete'),
 );
 
+//Request Tracker Ticket Types
+define('RTT_OFFER', 1);
+define('RTT_DOCS', 2);
+define('RTT_FAULT', 3);
+define('RTT_INST', 4);
+define('RTT_MOD', 5);
+define('RTT_START', 6);
+define('RTT_STOP', 7);
+define('RTT_REMOVE', 8);
+define('RTT_OTHER', 9);
+
+
+$RTT_TYPES = array(
+    RTT_OFFER => array(
+        'label' => trans('Offer'),
+        'color' => 'red',
+        'background' => 'black',
```

Comment:
A skoro już takie coś wchodzi to może zrobić to jako odwołanie do klasy css, czyli po prostu 'class' => 'lms-ui-rt-ticket-type-....'?